libs/crystal.json

```diff
@@ -0,0 +1,13 @@
+[
+  {"id": "cr.kemal", "imports": ["kemal"], "name": "kemal", "repo": "kemalcr/kemal", "tags": ["framework"], "tech": ["app-arch"], "status": "model-ready"},
+  {"id": "cr.amber", "imports": ["amber"], "name": "amber", "repo": "amberframework/amber", "tags": ["framework"], "tech": ["app-arch"], "status": "model-ready"},
+  {"id": "cr.lucky", "imports": ["lucky"], "name": "lucky", "repo": "luckyframework/lucky", "tags": ["framework"], "tech": ["app-arch"], "status": "model-ready"},
+  {"id": "cr.db", "imports": ["db"], "name": "crystal-db", "repo": "crystal-lang/crystal-db", "tags": [""], "tech": ["app-blocks"], "status": "model-ready"},
```

Comment:
I think an empty string is a bad idea for the tags field. Could you please add tags from [this list](https://github.com/sourcerer-io/awesome-libraries/blob/master/technologies.json) or create your own tag, like 'db-api'. Don't forget to run technologies.js when you add new tags.